app/src/layouts/Layout.vue

```diff
@@ -21,6 +21,15 @@
           <menu-drawer />
         </q-drawer>
         <router-view />
+        <q-footer class="lt-md">
+          <q-toolbar>
+            <q-tabs breakpoint="1024" class="full-width">
+              <q-route-tab :to="{ name: 'Group', params: { code: groupCode }}" name="home" icon="home" label="Home" />
```

Comment:
Probably you just let that for later but just in case: labels should be translated using the function `t` from the composable `useI18n`. Check for example the `/components/DeleteBtn.vue` to see an example of use. Then the words should be added to the lang files.